### DISCUSSION TITLE
Authorization header in KurentoClient

### DISCUSSION THREAD

**Discussion by ttiagojm** [Kurento/kurento#77] [Community]
I saw [here](https://groups.google.com/g/kurento/c/0V01yr6zb2M?pli=1) that in 2016 is not possible. But can anyone tell me if in current version is it possible to pass Authorization header to send a bearer token for authentication? This when the HTTP request sends connection upgrade. 

If it is not possible there are any alternatives using KurentoClient java package?

**Comment by slabajo** [comment]
It is not currently possible, although it is an interesting feature, that we take note in our roadmap. Hoepfully we can make it in a near future version.
